src/web/app/pages-instructor/instructor-course-student-details-page/instructor-course-student-details-page.component.ts

```diff
@@ -21,18 +22,25 @@ interface StudentDetails {
   styleUrls: ['./instructor-course-student-details-page.component.scss'],
 })
 export class InstructorCourseStudentDetailsPageComponent implements OnInit {
-
   user: string = '';
   student?: StudentAttributes;
   studentProfile?: StudentProfile;
+  photoUrl: string = '';
+
+  private backendUrl: string = environment.backendUrl;
```

Comment:
This can be a local variable?

Reply:
Yup updated. I have just used `environment.backendUrl` directly in the profile picture link.

I have also changed all unnecessary uses of this variable in other classes.